readme.md

```diff
@@ -8,28 +8,46 @@
 
 ## Overview
 
+Chief-of-state (COS) is a dockerized persistence framework for building modern, clustered, reactive applications.
+COS encourages best practices in CQRS and event-sourcing through simple, language-agnostic interfaces via gRPC, and it
+allows developers to describe their schema with Protobuf. Under the hood, COS leverages Akka to scale out and guarantee
+performant, reliable persistence.
+
+Chief-of-state was built by Namely with the following principles:
+* Wire format should be the same as persistence (Protobuf is a better schema definition than DDL)
+* Prototypes should scale
```

Comment:
How about: Scaling should not require re-architecture

Reply:
Yeah i was trying to accomplish 2 thoughts here:
- COS lets you prototype quickly
- You don't need to re-architect the app to scale